Simplenote/src/main/java/com/automattic/simplenote/Simplenote.java

```diff
@@ -49,7 +50,7 @@ public void onCreate() {
         super.onCreate();
 
         CrashUtils.initWithContext(this);
-
+    
```

Comment:
```suggestion

```

This looks like some unnecessary spaces might have been accidentally added.  We can remove them.